### PR TITLE
[NLP-157] [NLP-142] Sanity webhook for retraining, and endpoint for retraining on CSV

### PR DESCRIPTION
**Pull request by rynoV**
Builds on model retrain code from #83

### CODE REVIEW

**Review by JCayabyab — APPROVED** [review]
Does this PR need to merged with/before/after #83 ?

**Comment by rynoV** [comment]
> Does this PR need to merged with/before/after #83 ?

@JCayabyab I think it will be best for these changes to be merged into #83 as it adds some stuff related to creating the model.

File: server/schemas/submit.py [hunk 1]
```diff
@@ -61,3 +61,17 @@ class SubmitOut(BaseModel):
...
+    deleted: List[str]
+    updated: List[str]
+    all: List[str]
+
+
+class SanityUpdate(BaseModel):
+  transactionId: str
+  projectId: str
+  dataset: str
+  ids: SanityUpdateIds
```
Comment (JCayabyab) [inline server/schemas/submit.py:77]: newline end of this file

**Review by JCayabyab — APPROVED** [review]
looks good, one extremely nitpicky formatting comment then you can merge

**Pull request merged**